go/pkg/pwapi/api_season-list.go

```diff
@@ -0,0 +1,14 @@
+package pwapi
+
+import (
+	"context"
+
+	"pathwar.land/pathwar/v2/go/pkg/pwdb"
+)
+
+func (svc *service) SeasonList(ctx context.Context, in *SeasonList_Input) (*SeasonList_Output, error) {
+	var ret SeasonList_Output
+	svc.db.Where(pwdb.Season{IsGlobal: true, IsTesting: false, Status: 1}).Find(&ret.Items)
```

Comment:
```suggestion
	svc.db.Where(pwdb.Season{IsTesting: false, Status: 1}).Find(&ret.Items)
```

`IsGlobal` is limited to the default season, the "global season".